docs/guidance/ingress.md

```diff
@@ -70,6 +73,61 @@ kubectl describe ingress raycluster-ingress-head-ingress
 #               will report "404 Not Found".
 ```
 
+### Example: Manually setting up NGINX Ingress on KinD
+```sh
+# Step 1: Create a KinD cluster with `extraPortMappings` and `node-labels`
+cat <<EOF | kind create cluster --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
+EOF
+
+# Step 2: Install NGINX ingress controller
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
+kubectl wait --namespace ingress-nginx \
+  --for=condition=ready pod \
+  --selector=app.kubernetes.io/component=controller \
+  --timeout=90s
+
+# Step 3: Install KubeRay operator
+pushd helm-chart/kuberay-operator
+helm install kuberay-operator .
+
+# Step 4: Install RayCluster and create an ingress separately. 
+popd
+kubectl apply -f ray-operator/config/samples/ray-cluster.separate-ingress.yaml
+
+# Step 5: Check ingress created by Step 4.
+kubectl describe ingress raycluster-ingress-head-ingress
+
+# [Example]
+# ...
+# Rules:
+# Host        Path  Backends
+# ----        ----  --------
+# *
+#             /raycluster-ingress/(.*)   raycluster-ingress-head-svc:8265 (10.244.0.11:8265)
+# Annotations:  nginx.ingress.kubernetes.io/rewrite-target: /$1
+
+# Step 6: Check `<ip>/raycluster-ingress/` on your browser. You will see the Ray Dashboard.
+#        [Note] The forward slash at the end of the address is necessary. `<ip>/raycluster-ingress`
+#               will report "404 Not Found".
+```
+
```

Comment:
Add the link (https://kind.sigs.k8s.io/docs/user/ingress/) in both the PR description and `ingress.md` to help users resolve issues by themselves.